_data/projects/qeda.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: Quantum Electronics Design Automation!
+desc: QEDA allows you to design and build real, affordable, room-temperature stable, and scalable quantum hardware at home!
+tags:
+  - Quantum
+  - Electronics
+  - EDA
+  - Automation
+  - Python
+  - Open Source
+  - Beginner Friendly
+  - Documentation
+  - Compiler Development
+  - Circuit Synthesis
+upforgrabs:
+  name: good first issue
+  link: https://github.com/Spooky-Manufacturing/QEDA/labels/good%20first%20issue
```

Comment:
@NoahGWood Please go through the errors/suggestions as described by our bot in this PR. we need that `site` field which should be the repo or homepage link

Reply:
will do